### DISCUSSION TITLE
Displacement in Houdini

### DISCUSSION THREAD

**Discussion by BrianHanke** [dreamworksanimation/openmoonray#96] [Q&A] [answered]
Just checking: since the Render Geometry Settings node is broken in MoonRay there's no way to set tessellation level in Solaris, right?

**Comment by scegielski** [comment, marked as answer]
This should work in a LOP wrangle node:
```
usd_addprimvar(0, @primpath, "moonray:mesh_resolution", "float");
usd_setprimvar(0, @primpath, "moonray:mesh_resolution", 10));
```
Make sure the attribute gets added to the objects in the scene graph details pane.
